server/etcdserver/api/membership/cluster_test.go

```diff
@@ -1049,10 +1049,9 @@ func TestAddMemberSyncsBackendAndStoreV2(t *testing.T) {
 			backendMembers: []*Member{alice},
 		},
 		{
-			name:           "Adding member should fail if it exists in both",
+			name:           "Adding member should success if it exists in both",
```

Comment:
This test case was meant to prevent same member being added twice, while we are fixing the issue of v3store rejecting learner being promoted. Should the test be updated to separate those two cases?

Reply:
For v3store (bbolt), when we add a member or promote a learner, etcd just marshals the member and write the data into bbolt, if it already exists, just overwrites it. This is the behaviour in both release-3.4, release-3.6 and main. But In release-3.5, it returned an error already exists, it's a bug and now we fix it in this PR.

Also FYI.
- When we add a member again, the `ValidateConfigurationChange` will fail.
- We also have separate unit test case to verify the v2store will return error when adding a member again. (see line 1057 in cluster_test.go).

Reply:
> Should the test be updated to separate those two cases?

No need. Please see my response above.